be/src/olap/rowset/segment_v2/indexed_column_reader.cpp

```diff
@@ -31,12 +31,12 @@ Status IndexedColumnReader::load(bool use_page_cache, bool kept_in_memory) {
     _use_page_cache = use_page_cache;
     _kept_in_memory = kept_in_memory;
 
-    _type_info = get_type_info((FieldType)_meta.data_type());
+    _type_info = get_scalar_type_info((FieldType)_meta.data_type());
```

Comment:
question: can we make sure that the type here is scalar type?

Reply:
because array, map and struct cannot be got only with FiedType info